windows/utils/win_get_os_version.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Get guest OS version with major and minor build number.
```

Comment:
When will this utility be used?

Reply:
You can use this task to get OS build number before and after Windows updates install to see if build number changed.